src/components/UI/Alert/Alert.js

```diff
@@ -0,0 +1,15 @@
+import { ToastContainer, toast } from 'react-toastify';
```

Comment:
Suggestion: since it is toast you could give file name as Toast.js

Reply:
Actually, the code i already pushed and file name doesn't get updated in Github this might cause unnecessary issues so i guess alert is also politically correct